### DISCUSSION TITLE
Aspects between transiting planets and natal planets

### DISCUSSION THREAD

**Discussion by obertetti** [g-battaglia/kerykeion#89] [General]
Hi @g-battaglia, love your project, thanks so much for your hard work on this!

Once you have generated today's transit through the AstrologicalSubject function, how can you assess the aspects between transiting planets and natal planets (e.g transiting Jupiter trine natal Sun)?

I've tried using the NatalAspects function applied to today's transit but doesn't seem to be generating the right results.

Please see code below.

```
from kerykeion import AstrologicalSubject, KerykeionChartSVG
subject = AstrologicalSubject("Kanye", 1977, 6, 8, 8, 45, "Atlanta", "USA")
transits_today = AstrologicalSubject(name = "Kanye", city="Atlanta", nation = "USA") 

aspects_today = NatalAspects(transits_today)
relevant_aspects= aspects_today.relevant_aspects
```

**Comment by g-battaglia** [comment]
You should do it like this:
```python
from kerykeion import AstrologicalSubject, SynastryAspects

subject = AstrologicalSubject("Kanye", 1977, 6, 8, 8, 45, "Atlanta", "USA")
transits_today = AstrologicalSubject(name = "Kanye", city="Atlanta", nation = "USA") 

aspects_today = SynastryAspects(subject, transits_today)
relevant_aspects= aspects_today.relevant_aspects

print(relevant_aspects)
```

**Reply by obertetti** [reply]
brilliant, thank you so much for your response!
